web_src/js/components/RepoActionView.vue

```diff
@@ -297,6 +304,10 @@ const sfc = {
         body,
       });
     },
+
+    isDone(status) {
+      return ['success', 'skipped', 'failure', 'cancelled'].includes(status);
```

Comment:
Just a note. Containment checks in collections are better done using `Set` and `set.has` because those are O(1) compared to array access which is O(n). In this case it does not matter as the number is small, so I'm ok with it.